core/src/main/java/com/novoda/noplayer/drm/DownloadDrmSessionCreator.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.noplayer.drm;
+
+import com.google.android.exoplayer2.drm.DrmSessionManager;
+import com.google.android.exoplayer2.drm.FrameworkMediaCrypto;
+
+import java.util.UUID;
+
+public class DownloadDrmSessionCreator implements DrmSessionCreator {
```

Comment:
This can be a package-private class - only the interface should be public here really